AY2021S2-CS2103-T14-1/tp
# Add Alias implementation details to DG

### What this does

This PR adds design details and diagrams of the command history subsystem to the DG.

Closes #113

### How to test

1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Skim through the Alias subsection of the Implementation section.
5. See if the explanations and diagrams are clarifying.
